cms/static/sass/views/_container.scss

```diff
@@ -331,3 +331,112 @@
     }
   }
 }
+
+
+.move-xblock-modal {
+
+  button {
+    background: transparent;
+    border-color: transparent;
+    padding: 0;
+    border: none;
+  }
+
+  .breadcrumb-container {
+    margin-bottom: ($baseline/4);
+    border: 1px solid #D9D9D9;
+    padding: ($baseline/2);
+    background: #F2F8FB;
```

Comment:
I echo @muzaffaryousaf's comment above which is that we should always use variables for colors.